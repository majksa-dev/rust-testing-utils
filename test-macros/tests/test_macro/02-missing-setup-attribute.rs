use testing_utils::macros as utils;

fn teardown(ctx: usize) {
    println!("Tearing down: {}", ctx);
}

#[utils::test(teardown = teardown)]
fn single_server(_ctx: &mut usize) {}

fn main() {}
