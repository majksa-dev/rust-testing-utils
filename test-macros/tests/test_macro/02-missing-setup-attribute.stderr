error: setup attribute not found
 --> tests/test_macro/02-missing-setup-attribute.rs:7:1
  |
7 | #[utils::test(teardown = teardown)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `utils::test` (in Nightly builds, run with -Z macro-backtrace for more info)
